packages/web/src/components/coinflow-onramp-modal/CoinflowOnrampModal.tsx

```diff
@@ -46,6 +46,11 @@ export const CoinflowOnrampModal = () => {
     }
   }, [serializedTransaction])
 
+  const handleClose = useCallback(() => {
```

Comment:
May want to check that we have similar logic in the mobile web and native mobile drawers? They handle close differently.